File: app/src/main/java/com/paxees/tcc/network/networkmodels/response/models/MyAddressesListResponse.java

package com.paxees.tcc.network.networkmodels.response.models;

import java.io.Serializable;
import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class MyAddressesListResponse implements Serializable
{

    @SerializedName("billing")
    @Expose
    private List<Billing> billing = null;
    @SerializedName("shipping")
    @Expose
    private List<Shipping> shipping = null;
    private final static long serialVersionUID = 1365546103307950070L;

    public List<Billing> getBilling() {
        return billing;
    }

    public void setBilling(List<Billing> billing) {
        this.billing = billing;
    }

    public List<Shipping> getShipping() {
        return shipping;
    }

    public void setShipping(List<Shipping> shipping) {
        this.shipping = shipping;
    }

}

File: app/src/main/java/com/paxees/tcc/network/networkmodels/response/models/MyAddressesListResponse.kt
package com.paxees.tcc.network.networkmodels.response.models


import com.google.gson.annotations.SerializedName

class MyAddressesListResponse {
    @SerializedName("avatar_url")
    var avatarUrl: String = ""
    @SerializedName("billing")
    var billing: Billing = Billing()
    @SerializedName("date_created")
    var dateCreated: String = ""
    @SerializedName("date_created_gmt")
    var dateCreatedGmt: String = ""
    @SerializedName("date_modified")
    var dateModified: String = ""
    @SerializedName("date_modified_gmt")
    var dateModifiedGmt: String = ""
    @SerializedName("email")
    var email: String = ""
    @SerializedName("first_name")
    var firstName: String = ""
    @SerializedName("id")
    var id: Int = 0
    @SerializedName("is_paying_customer")
    var isPayingCustomer: Boolean = false
    @SerializedName("last_name")
    var lastName: String = ""
    @SerializedName("_links")
    var links: Links = Links()
    @SerializedName("meta_data")
    var metaData: List<MetaData> = listOf()
    @SerializedName("role")
    var role: String = ""
    @SerializedName("shipping")
    var shipping: Shipping = Shipping()
    @SerializedName("username")
    var username: String = ""
}